bin/oref0-setup.sh

```diff
@@ -232,6 +232,13 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
             echo
         fi
     fi
+    read -p "Would you like to [D]ownload precompiled go pump communication library or build them from [S]ource? [D]/S " -r
+    buildgofromsource=false
+    if [[ $REPLY =~ ^[Ss]$ ]]; then
+      buildgofromsource=true
+      echo "Building go libs from source"
+    fi
```

Comment:
Should probably have an else clause here acknowledging when someone selects Download.